README.md

```diff
@@ -44,16 +40,16 @@ your local Maven repository, and runs all of the unit tests:
 The project can be built and deployed on a variety of runtime platforms, but if you want to see it in 
 action as quickly as possible try this:
 
-    $ mvn clean install -Pinstall-all-wildfly8
-    $ cd tools/server-all/target/wildfly-8.2.0.Final/
+    $ mvn clean install -Pinstall-all-wildfly9
```

Comment:
@EricWittmann Happy to do either. Especially as we'll be upgrading to a version of KC with fixed relative URL handling.